src/processors/oeth/utils.ts

```diff
@@ -63,16 +63,20 @@ export async function createRebaseAPY(
 
   // get last APY to compare with current one
   let lastApy =
-    apies.slice(apies.length - 2).find((apy) => apy.id < dateId) ??
+    apies.find((apy) => apy.id < dateId) ??
```

Comment:
Are we sure we want this change? If we have anything in the apies array it will definitely be recent and we wouldn't care about anything in there less than 2 instances past.

Reply:
i might be wrong